pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools<69.3", "wheel"]
+requires = ["setuptools", "wheel"]
```

Comment:
it would probably be wise to set a lower bound then, since they introduce a backward incompatible change

and related: does pylake not have a CI workflow to check if the project is packaged correctly (before attempting a release)?